nbqa/config_parser.py

```diff
@@ -96,6 +96,13 @@ def _parse_setupcfg_or_toxini_config(
     (".nbqa.ini", _parse_nbqa_ini_config),
 ]
 
+DEFAULT_CONFIG: Mapping[str, Mapping] = {
```

Comment:
I will be simpler to store these configurations in `default.toml` and parse the config using existing `parse_from_pyproject_toml`. Then we can use `config.merge(default_config)`. This will be much simpler.

Its better not to have configuration as part of source code

Reply:
yes, you're right :+1: I'll do that in the next iteration

Reply:
I haven't figured out how to package an extra toml file yet, but in the meantime, does the rest look alright to you?